nrf70_bm_lib/docs/source/nrf70_bm_porting_guide.rst

```diff
@@ -89,6 +89,36 @@ For more information, visit the `nRF Connect SDK nrfxlib repository <https://git
 
    The git submodule pulls in the entire repo, but only `nrf_wifi` directory is required for the BM driver.
 
+TX power configuration
+**********************
+
+The maximum transmit power achieved on an nRF70 Series device-based product depends on the frequency band and operating channel.
+This varies across different Printed Circuit Board (PCB) designs.
+
+Multiple calibrations and checks are implemented to ensure consistency across channels and devices.
+However, these values depend on PCB design, which may result in Error Vector Magnitude (EVM) and spectral mask failures.
+To avoid this problem, you can specify the power ceiling at which the EVM and spectral mask are met for a given PCB design.
+
+The build-time parameters are made available to drivers through the `C` header file.
+The reference header files are located in the `nrf70_bm_lib/include` directory.
+
+The following files are used to configure the TX power:
+
+- `nrf70_tx_pwr_ceil_dk.h`: Contains the TX power configuration for the nRF7002 Development Kit (DK).
+
+- `nrf70_tx_pwr_ceil_ek.h`: Contains the TX power configuration for the nRF7002 Evaluation Kit (EK).
+
+For a new platform, you can create a new header file with the same structure as the reference files and replace the values with the appropriate values for the new platform.
```

Comment:
I think, here, we need to say one sentence about HOW a developer will determine the appropriate values for the new platform. Just a single informative message, highlighting what needs to be done,